debian/snapd.install

```diff
@@ -2,3 +2,4 @@ usr/bin/snap
 usr/bin/snap-exec /usr/lib/snapd/
 usr/bin/snapctl
 usr/bin/snapd /usr/lib/snapd/
+usr/lib/snapd/system-shutdown
```

Comment:
I think this should be in the part that gets installed in core images (perhaps another package, perhaps one of the existing packages). At the very least the unit file should never be installed on regular systems.